schemas/pluginSchema.json

```diff
@@ -11,6 +11,7 @@
             "type": "string"
         },
         "namespace": {
+            "deprecated": true,
```

Comment:
I would not add this line for now. We got feedback before about this that a deprecated field will be marked by VSC.

Reply:
Updated, thanks!